examples/cli-typer-base/main.py

```
@@ -13,69 +13,69 @@
-    """Retrieves the values from an api
-    https://bluelytics.com.ar/#!/api and render the data in a table using rich.
-    """
-    exchange = None
-    for i in track(range(3), description="Fetching..."):
-        exchange = Exchange()
-    options = (euro, official)
+   """Retrieves the values from an api
+   https://bluelytics.com.ar/#!/api and render the data in a table using rich.
+   """
+   exchange = None
+   for i in track(range(3), description="Fetching..."):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)